examples/angular/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "angular",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "serve:ssr:angular": "node dist/angular/server/server.mjs",
+    "serve:ssr:angular-ssr-prefetch": "nodemon dist/angular/server/server.mjs",
+    "analyze-bundle": "ng build --configuration=analyze-bundle && source-map-explorer dist/**/*.js --no-border-checks --html result.html"
+  },
+  "private": true,
+  "dependencies": {
+    "@adyen/adyen-web": "^6.0.0-beta.0",
```

Comment:
Maybe to use `"@adyen/adyen-web": "^6.0.0"`?

Reply:
True. I will update it!